src/telemetryClient.ts

```diff
@@ -15,12 +15,19 @@ const aiKey: string = packageJSON.aiKey;
 export class TelemetryClient {
     public static initialize(context: vscode.ExtensionContext){
         this._extensionContext = context;
+        let iotWorkbenchExt = vscode.extensions.all.find((element) => element.id === "vsciot-vscode.vscode-iot-workbench");

```

Comment:
You could use getExtension api: https://code.visualstudio.com/docs/extensionAPI/vscode-api#extensions.getExtension